en/docs/get-started/about-this-release.md

```diff
@@ -398,14 +398,54 @@ The following mediators and artifacts are removed from the ESB runtime in all ve
 
 Streaming Integrator serves the streaming-based integrations and acts as the event stream provider for Streaming APIs in the data plane of the API Manager platform. This is a newly added component into WSO2 API Manager 4.0.0.
 
-It is now available to download from [here](https://wso2.com/api-management/#).
+It is now available to download from [here](https://github.com/wso2/streaming-integrator/releases/download/v4.0.0/wso2si-4.0.0.zip).
 
 ### New features
 
-- [Deploy AsyncAPI definitions to the WSO2 API-M Service Catalog]({{base_path}}/use-cases/streaming-usecase/exposing-stream-as-managed-api-in-service-catalog/)
-- Support to capture multiple operations from a single CDC source
-- RegEx matching dynamic in copy function
-- SSE source functionality
+- **[Deploy AsyncAPI definitions to the WSO2 API-M Services]({{base_path}}/use-cases/streaming-usecase/exposing-stream-as-managed-api-in-service-catalog/)**
+    
+	 WSO2 Streaming Integrator has the inbuilt capability to directly deploy siddhi applications with Async Api definition to WSO2 Api Manager services which afterwards can be used to deploy an api.
+
+- **SSE Sink / Source support over http**
+    
+	 SSE is an HTTP-based protocol that allows one-way communication similar to WebHooks, from the server to the client. For more information , see [SSE Server Sink](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#sse-server-sink) , [SSE Source](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#sse-source).
+	 
+- **WebSub Sink / Source support over http**
+    
+	 Implementation is based on the [WebSub](https://www.w3.org/TR/websub/) protocol specification.WebSubs only allows one-way communication, from a caller web app to a callee web app. For more information , see [WebSub Sink](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#websubhub-sink) , [WebSub Source](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#websubhub-source).
+
+- **Improved MongoDB Store**
+    
+	 Several improvements are done like allow encryption protocol to be configurable, Object_ID support when joining and other bug fixes. For more information , see [Mongo DB Store](https://siddhi-io.github.io/siddhi-store-mongodb/api/2.1.2/#mongodb-store).
```

Comment:
'Object_ID-support when joining' what?
Consider changing it to something like - 'The Object_ID can be used to .... when .. joins ....'

"The improvements include the following: Configurable encryption protocol, Object_ID-support when joining, and other bug fixes. For more information, see [Mongo DB Store](https://siddhi-io.github.io/siddhi-store-mongodb/api/2.1.2/#mongodb-store)."

Reply:
improved